service/migration_manager.cc

```diff
@@ -970,7 +970,6 @@ future<> migration_manager::announce_with_raft(std::vector<mutation> schema, gro
         },
         guard, std::move(description));
 
-    // TODO: no retry if group0_concurrent_modification, add it?
```

Comment:
how is this related to the current patch?

Reply:
Removed.